### PR TITLE
GlusterFS reconfig procedure

### PR DESCRIPTION
**Pull request by mramendi**

### CODE REVIEW

File: docs/product-assembly_pam-on-openshift/src/main/asciidoc/glusterfs-reconfig-proc.adoc [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+.Procedure
+
+. Run the following command:
++
+[subs="verbatim,macros"]
+----
+oc get storageclass 
+----
++
+In the results, check whether the `(default)` marker is on the storage class that lists `glusterfs`. For example, in the following output the default storage class is `gluster-container`, which does list `glusterfs`:
```
Comment (michelehaglund) [inline docs/product-assembly_pam-on-openshift/src/main/asciidoc/glusterfs-reconfig-proc.adoc]: Is it a "marker" of simply a code line?

File: docs/product-assembly_pam-on-openshift/src/main/asciidoc/glusterfs-reconfig-proc.adoc [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+----
++
+. Edit the `storage_config.yaml` file:
+.. Remove the lines with the following keys:
+** `creationTimestamp`
+** `resourceVersion`
+** `selfLink`
+** `uid`
+.. On the line with the `volumeoptions` key, add the following two options: `features.cache-invalidation on, performance.nl-cache on`
+. Remove the existing default storage class. Run the following command:
```
Comment (michelehaglund) [inline docs/product-assembly_pam-on-openshift/src/main/asciidoc/glusterfs-reconfig-proc.adoc]: Maybe reword for consistency:
Run the following command to remove the existing default storage class:

File: docs/product-assembly_pam-on-openshift/src/main/asciidoc/glusterfs-reconfig-proc.adoc [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+----
++
+Where `class-name` is the name of the default storage class. For example:
++
+[subs="verbatim,macros"]
+----
+oc delete storageclass gluster-container
+----
++
+. Re-create the storage class using the new configuration. Run the following command:
```
Comment (michelehaglund) [inline docs/product-assembly_pam-on-openshift/src/main/asciidoc/glusterfs-reconfig-proc.adoc]: Maybe reword for consistency:
Run the following command to re-create the storage class using the new configuration:

File: docs/product-assembly_pam-on-openshift/src/main/asciidoc/glusterfs-reconfig-proc.adoc [hunk 4]
```diff
@@ -0,0 +1,66 @@
+[id='glusterfs-reconfig-proc']
+= Changing GlusterFS configuration
+
+You must check whether your OpenShift environment uses GlusterFS to provide permanent storage volumes. If it uses GlusterFS, you must change the GlusterFS configuration.
```
Comment (bmozaffa) [inline docs/product-assembly_pam-on-openshift/src/main/asciidoc/glusterfs-reconfig-proc.adoc]: you must change the GlusterFS configuration -> you should tune your GlusterFS storage

File: docs/product-assembly_pam-on-openshift/src/main/asciidoc/glusterfs-reconfig-proc.adoc [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+oc get storageclass gluster-container -o yaml >storage_config.yaml
+----
++
+. Edit the `storage_config.yaml` file:
+.. Remove the lines with the following keys:
+** `creationTimestamp`
+** `resourceVersion`
+** `selfLink`
+** `uid`
+.. On the line with the `volumeoptions` key, add the following two options: `features.cache-invalidation on, performance.nl-cache on`
```
Comment (bmozaffa) [inline docs/product-assembly_pam-on-openshift/src/main/asciidoc/glusterfs-reconfig-proc.adoc]: I would show them what the line would then look like, for example:

`volumeoptions: client.ssl off, server.ssl off, features.cache-invalidation on, performance.nl-cache`
